src/main/java/com/opentable/metrics/DefaultMetricsConfiguration.java

```diff
@@ -73,6 +87,14 @@ public MetricSetBuilder metricSetBuilder(MetricRegistry registry) {
         return builder;
     }
 
+    /**
+     * Create an application event listener that listens to every application event
+     * if there is a metric set builder that is waiting for an event to be registered,
+     * we'll build and register it when that event happens
+     *
+     * @param registry metric register to register the metrics
```

Comment:
"registry"

Reply:
This is a very philosophical statement: "registry metric register to register the metrics" :-)

Reply:
Will fix